examples/magento-graphcms/components/Layout/LayoutFull.tsx

```diff
@@ -89,6 +95,13 @@ export function LayoutFull(props: LayoutFullProps) {
             >
               <Trans>Customer Service</Trans>
             </MenuFabSecondaryItem>,
+            <MenuFabSecondaryItem
+              key='wishlist'
+              icon={<IconSvg src={iconHeart} size='medium' />}
+              href='/wishlist'
```

Comment:
Can we add a Dot badge https://mui.com/components/badges/#dot-badge when Wishlist has items